### PR TITLE
Added Abbrevations

### PR DESCRIPTION
**Pull request by Judeitopa**
Closes #

- Added new abbreviation
- dgaf.json

### CODE REVIEW

File: public/server/db/ui.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "definition": "User Interface"
```
Comment (Njong392) [inline public/server/db/ui.json:2]: Abbreve is an unofficial dictionary for slang. Unfortunately, abbreviations like these don't fit in. Hope you understand.

File: public/server/db/ux.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "definition": "User Experience"
```
Comment (Njong392) [inline public/server/db/ux.json:2]: Abbreve is an unofficial dictionary for slang. Unfortunately, abbreviations like these don't fit in. Hope you understand.

File: public/server/db/dgaf.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "definition": "Don't Give A Fuck"
```
Comment (Njong392) [inline public/server/db/dgaf.json:2]: This however looks really good!

Comment (Judeitopa) [reply public/server/db/dgaf.json:2]: @Njong392 ok. will correct it now

**Review by Njong392 — APPROVED** [review]
Thank you for the corrections made.

**Pull request merged**